app/scripts/untracked_files

```diff
@@ -45,7 +45,7 @@ $ignored_files = [
     'privacy/notices/firefox-hello.lang',
     'privacy/notices/firefox.lang',
 ];
-$ignored_locales = ['ja'];
+$ignored_locales = ['hi', 'ja'];
```

Comment:
We don't support `hi`, we support `hi-IN`. But there's an activation file for legal-docs for the former.
https://github.com/mozilla-l10n/www.mozilla.org/tree/master/hi